com.microsoft.mrtk.graphicstools.unity/Samples~/MaterialGallery/Materials/ShaderBallNormalMap.mat

```diff
@@ -9,8 +9,8 @@ Material:
   m_PrefabAsset: {fileID: 0}
   m_Name: ShaderBallNormalMap
   m_Shader: {fileID: 4800000, guid: c331f6c43a2ef0945864cb668f2653c9, type: 3}
-  m_ShaderKeywords: _DIRECTIONAL_LIGHT _HOVER_LIGHT _NORMAL_MAP _REFLECTIONS _SPECULAR_HIGHLIGHTS
-    _USE_WORLD_SCALE
+  m_ShaderKeywords: _DIRECTIONAL_LIGHT _EMISSION _HOVER_LIGHT _NORMAL_MAP _REFLECTIONS
```

Comment:
Did you mean to add emission to this material? I noticed the emissive color is black so it's not doing anything.

Reply:
Leftover from previous attempt, unticked.